src/pokedex_plus_hgss.c

```diff
@@ -6627,6 +6628,12 @@ static void PrintEvolutionTargetSpeciesAndMethod(u8 taskId, u16 species, u8 dept
             CopyItemName(item, gStringVar2);
             StringExpandPlaceholders(gStringVar4, sText_EVO_ITEM_COUNT_999);
             break;
+        case EVO_DEFEAT_WITH_ITEM:
+            item = evolutions[i].param;
+            CopyItemName(item, gStringVar2);
+            StringCopy(gStringVar3, GetSpeciesName(species));
+            StringExpandPlaceholders(gStringVar4, sText_EVO_ITEM_COUNT_999);
```

Comment:
```suggestion
            StringExpandPlaceholders(gStringVar4, sText_EVO_DEFEAT_WITH_ITEM);
```

Reply:
Absolutely brilliant oversight, thanks for noticing